rpcserver.go

```diff
@@ -2354,10 +2353,10 @@ func (state *gbtWorkState) updateBlockTemplate(s *rpcServer, useCoinbaseValue bo
 	// generated.
 	var msgBlock *wire.MsgBlock
 	var targetDifficulty string
-	latestHash, _ := s.server.blockManager.chainState.Best()
+	best := s.server.blockManager.chain.BestSnapshot()
```

Comment:
Same here regarding `latestHash := &s.server.blockManager.chain.BestSnapshot().Hash`

Reply:
`minimumMedianTime` requires `best` as a param on line 2400, keeping this as a result.

Reply:
Ok, I thought that was in another func when reviewing the diff.